utils/check-credentials.js

```diff
@@ -0,0 +1,26 @@
+const { askYesNoQuestion } = require('./readline');
+
+module.exports = async function checkCredentials(
+    accountId,
+    networkId,
+    keyStore
+) {
+    if (!(await keyStore.getKey(networkId, accountId))) {
+        console.log(
+            `Unable to find [ ${networkId} ] credentials for [ ${accountId} ]`
+        );
+        const defaultKeyPair = await keyStore.getKey('default', accountId);
```

Comment:
since `networkId` can be either `testnet` or `default` should you check if the key is in the other location? like

```js
const otherNetworkId = networkId === 'default ? 'testnet' : 'default';
const defaultKeyPair = await keyStore.getKey(otherNetworkId, accountId);
```